orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/job/WaitOnRemoteJobStageCompletion.java

```diff
@@ -0,0 +1,23 @@
+package com.netflix.spinnaker.orca.clouddriver.tasks.job;
+
+import com.netflix.spinnaker.kork.annotations.Alpha;
+import com.netflix.spinnaker.orca.api.pipeline.TaskResult;
+import com.netflix.spinnaker.orca.api.pipeline.models.StageExecution;
+import org.springframework.stereotype.Component;
+
+import javax.annotation.Nonnull;
+
+@Component
+@Alpha
+public class WaitOnRemoteJobStageCompletion extends WaitOnJobCompletion {
+
+  @Nonnull
+  @Override
+  public TaskResult execute(@Nonnull StageExecution stage) {
+    return TaskResult
+        .builder(stage.getStatus())
+        .context(stage.getContext())
+        .outputs(stage.getOutputs())
```

Comment:
I'm confused why you're doing this. Code comment would be nice for sure, but I don't think we want to be copying context and output of a stage down to a task - that's backwards.

Reply:
Yea this approach was very naive the more I learn about Orca message handling - the stage status could be overridden by the last task status pretty easily.  I'm also going to leave context and outputs out of this monitor task regardless of the solution I settle on.